packages/atomic/cypress/integration/facets/category-facet.cypress.ts

```diff
@@ -347,9 +361,61 @@ describe('Category Facet with facetSearchEnable', () => {
       .should('be.visible');
   });
 
-  describe('When user starts typing on facet searchbox', () => {
-    it('Should display search results correctly');
-    it('Should trigger new filter when item is selected from search dropdown');
+  describe.skip('When user interacts with category facet searchbox', () => {
```

Comment:
This  should work now

Reply:
oh good to know, I will double check, if it works, I will enable it in this PR as well